dev/breeze/src/airflow_breeze/utils/path_utils.py

```diff
@@ -142,6 +142,12 @@ def reinstall_if_setup_changed() -> bool:
     except ModuleNotFoundError as e:
         if "importlib_metadata" in e.msg:
             return False
+        if "apache-airflow-breeze" in e.msg:
+            print(
+                """Missing Package`apache-airflow-breeze`.
+                   Use `pip install -e ./dev/breeze` to install the package."""
```

Comment:
```suggestion
                   Use `pipx install -e ./dev/breeze` to install the package."""
```

We are using `pipx` for breeze.